ansible_base/lib/abstract_models/common.py

```diff
@@ -62,10 +62,13 @@ def save(self, *args, **kwargs):
         '''
         update_fields = list(kwargs.get('update_fields', []))
 
-        if 'modified_by' not in update_fields:
+        if self.pk and 'modified_by' not in update_fields:
```

Comment:
I don't agree with adding `self.pk and ` here, and that might be the cause of some test failure. If creating a new object, that new object should have `modified_by` set. Contractually, this is hard to argue against.

Reply:
We can set it if there's no pk but we can't set update_fields in that case. If we set that Django treats it as an update and when there's no pk, there's nothing to update and we get an error.

Reply:
True. I remember how it worked before, at some point. By doing `update_fields = kwargs.get('update_fields', [])`, it made `update_fields` reference the list that existed inside of `kwargs`, if it existed there. If there was no `update_fields` in `kwargs`, then adding to the list would only add to the throw-away data structure which came from `[]`. The `list()` forces the rest of the code to be more explicit.

Reply:
Ok I've removed the self.pk check.

I don't want to remove the `list()`, because it's too magical/subtle what is going on there, relying on `update_fields` sometimes being an existing reference and sometimes being a new list.